dotnet/Getting_Started_VS/Getting-Started-VS/Program.cs

```diff
@@ -0,0 +1,168 @@
+﻿/*
+Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+SPDX-License-Identifier: Apache-2.0
+
+
+Purpose
+
+  This example verifies that you can connect to your AWS account and perform
+  simple read operations on your AWS objects. This example reads the names
+  of your Amazon RDS DB instances and your Amazon S3 buckets. If you don't
+  have any DB instances or buckets, the example still connects to your AWS
+  account and reports what it can't find.
+
+  This code example was created by using Visual Studio 2019.
+  This code example uses .NET Core 3.1 to create a cross-platform application.
+  This code example creates a console application.
+
+
+Prerequisites
+
+To build and run this example, you need the following:
+
+- The AWS SDK for .NET. For more information, see the AWS SDK for .NET
+  Developer Guide (https://docs.aws.amazon.com/sdk-for-net/v3/developer-guide/welcome.html).
+
+- AWS credentials and a default AWS Region. If you have the AWS CLI installed,
+  you can specify them in a local AWS config file such as C:\Users\username\.aws\config,
+  and an AWS credentials file such as C:\Users\username\.aws\credentials. For
+  more information, see the AWS Tools and SDKs Shared Configuration and
+  Credentials Reference Guide (https://docs.aws.amazon.com/credref/latest/refdocs/overview.html).
+
+- The AWSSDK.RDS package. This is already referenced in the solution.
+
+- The AWSSDK.S3 package. This is already referenced in the solution.
+
+
+Running the code
```

Comment:
Group this section and the next `Running the tests` under a parent Visual Studio section. Then also have instructions on how to build and test using the dotnet CLI.

It is actually very easy to do from the CLI. In the project directory execute the command `dotnet run` to build and run the project. From the test folder run `dotnet test` to run the tests.

Reply:
This is from our template - I have to have these (separate) sections. I can add the CLI instructions...

Reply:
...added the .NET CLI instructions